app/src/main/res/values/strings.xml

```diff
@@ -97,7 +97,6 @@
     <string name="error_answer_match">Answers do not match</string>
     <string name="error_img_preview">Can\'t load image preview</string>
     <string name="error_video_preview">Can\'t load video preview</string>
-    <string name="use_external_source">Use External Source</string>
```

Comment:
Do you also need to remove these for other languages?

Reply:
The other languages will be removed when I pull the latest strings from transifex. The idea is to only modify the main strings.xml and let transifex API modify the others, transifex overrides the translation files anyway.